test/test_images/latencymako/event_state.proto

```diff
@@ -0,0 +1,33 @@
+// Copyright 2019 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+package latencymako;
+
+import "google/protobuf/timestamp.proto";
+
+message EventsRecord {
+	map<string, google.protobuf.Timestamp> events = 1;
+}
+
+service EventsRecorder{
+	rpc RecordSentEvents(EventsRecord) returns (RecordReply) {}
+	rpc RecordAcceptedEvents(EventsRecord) returns (RecordReply) {}
+	rpc RecordFailedEvents(EventsRecord) returns (RecordReply) {}
+	rpc RecordReceivedEvents(EventsRecord) returns (RecordReply) {}
```

Comment:
I'll use an enum inside `EventsRecord` to distinguish the type of the record instead of that horror, promise.